spec/models/adviser_spec.rb

```diff
@@ -142,35 +144,18 @@
         adviser.run_callbacks(:commit)
       end
 
-      def queue_contains_a_job_for(job_class)
+      def aqueue_contains_a_job_for(job_class)
```

Comment:
Typo? Should this be `a_queue_contains_a_job_for` or just `queue_contains_a_job_for`?